### PR TITLE
Add documentation for using UI interpolation

### PR DESCRIPTION
**Pull request by jasonLaster**

### CODE REVIEW

File: docs/marionette.abstractview.md [hunk 1]
```diff
@@ -580,3 +581,35 @@ var MyView = Marionette.ItemView.extend({
 ```
 
 This applies to all view classes.
+
+## UI Interpolation
+
+Marionette UI offers a convenient way to reference jquery elements.
```
Comment (ahumphreys87) [inline docs/marionette.abstractview.md]: could we write 'jQuery'?

Man I hate being a nit pick :P

File: docs/marionette.abstractview.md [hunk 2]
```diff
@@ -580,3 +581,35 @@ var MyView = Marionette.ItemView.extend({
...
+UI elements can also be interpolated into event and region selectors.
+
+In this example, the buy button is referenced in a dom event and the checkout section is referenced in the region selector.
```
Comment (ahumphreys87) [inline docs/marionette.abstractview.md]: DOM

Sorry :(

**Comment by jasonLaster** [comment]
lol

**Comment by jasonLaster** [comment]
fixed

**Comment by ahumphreys87** [comment]
hehe thanks. :+1:

**Pull request merged**